assembl/static2/js/app/reducers/adminReducer/timeline.js

```diff
@@ -78,8 +103,17 @@ export const phasesInOrder: PhasesInOrderReducer = (state = List(), action) => {
   switch (action.type) {
   case CREATE_PHASE:
     return state.push(action.id);
-  case UPDATE_PHASES:
-    return List(action.phases.map(phase => phase.id));
+  case UPDATE_PHASES: {
+    const phases = action.phases.sort((a, b) => a.order - b.order);
+    return List(phases.map(phase => phase.id)); }
+  case MOVE_PHASE_UP: {
```

Comment:
You should not handle move up/down actions in this reducer, you should in fact remove the `InOrder` reducer.
Look at `voteSession.js`'s `voteProposalsById` reducer and `mapStateToProps` in `voteSessionAdmin.jsx` for an example (sort phases by order in `mapStateToProps`).